src/features/registry.js

```diff
@@ -12,6 +14,9 @@ systemJSPrototype.get = function (id) {
 };
 
 systemJSPrototype.set = function (id, module) {
+  if (!process.env.SYSTEM_PRODUCTION && !resolveIfNotPlainOrUrl(id, baseUrl)) {
```

Comment:
`new URL` is probably better since an absolute URL should be the input - `../asdf` isn't really a very sensible registry id either.